sessionctx/variable/sysvar.go

```diff
@@ -1475,7 +1487,8 @@ var defaultSysVars = []*SysVar{
 		atomic.StoreUint32(&config.GetGlobalConfig().Log.RecordPlanInSlowLog, uint32(tidbOptInt64(val, logutil.DefaultRecordPlanInSlowLog)))
 		return nil
 	}, GetSession: func(s *SessionVars) (string, error) {
-		return strconv.FormatUint(uint64(atomic.LoadUint32(&config.GetGlobalConfig().Log.RecordPlanInSlowLog)), 10), nil
+		enabled := atomic.LoadUint32(&config.GetGlobalConfig().Log.RecordPlanInSlowLog) == 1
+		return BoolToOnOff(enabled), nil
```

Comment:
This is technically a bug: a bool type should return the string value "ON/OFF". It is fixed here because the validation added converts it to ON/OFF anyway, which broke a test for this value.